typescript-sdk/apps/dojo/next.config.ts

```diff
@@ -16,7 +16,6 @@ const withMDX = createMDX({
 
 const nextConfig: NextConfig = {
   /* config options here */
-  output: "standalone",
```

Comment:
Why did we change this?

Reply:
I had added this before for the old contract testing, but since we're not deploying this, I can take it back out. It makes the startup process more consistent to not have this when linking in copilotkit local-dev versions.